ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/SubmitterInfo.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v2
+
+import com.daml.ledger.participant.state.v1.{ApplicationId, CommandId, Offset, Party}
+
+/** Information provided by the submitter of changes submitted to the ledger.
+  *
+  * Note that this is used for party-originating changes only. They are
+  * usually issued via the Ledger API.
+  *
+  * @param actAs the non-empty set of parties that submitted the change.
+  *
+  * @param applicationId an identifier for the DAML application that
+  *   submitted the command. This is used for monitoring and to allow DAML
+  *   applications subscribe to their own submissions only.
+  *
+  * @param commandId a submitter-provided identifier to identify an intended ledger change
+  *   within all the submissions by the same parties and application.
+  *
+  * @param deduplicationPeriod The deduplication period for the command submission.
+  *   If not given explicitly as an offset, the [[WriteService]] is responsible for
+  *   determining a suitable completion offset where deduplication starts according
+  *   to the [[ReadService]]'s deduplication guarantee.
```

Comment:
```suggestion
  *   Used for the deduplication guarantee described in the [[ReadService.stateUpdates]]
```

KV ledgers will ultimately translate the given information to record_time timestamps. So, its probably better to not be too specific here.

Reply:
:+1: